cli_reference/openshift_developer_cli/using_odo_in_a_restricted_environment/about-odo-in-a-restricted-environment.adoc

```diff
@@ -0,0 +1,19 @@
+[id="about-odo-in-a-restricted-environemnt"]
+= About {odo-title} in a restricted environemnt
+include::modules/openshift-developer-cli-attributes.adoc[]
+include::modules/common-attributes.adoc[]
+:context: about-odo-in-a-restricted-environemnt
+toc::[]
+
+
+To run `{odo-title}` in a disconnected {product-title} cluster or a cluster provisioned in a restricted environment, you must ensure that a cluster administrator has created a cluster with a mirrored registry.
+
+
+To start working in a disconnected cluster, you must first xref:../../../cli_reference/openshift_developer_cli/using_odo_in_a_restricted_environment/pushing-the-odo-init-image-to-the-restricted-cluster-registry.adoc#pushing-the-odo-init-image-to-a-mirror-registry_pushing-the-odo-init-image-to-the-restricted-cluster-registry[push the `odo init` image to the registry of the cluster] and then overwrite the `odo init` image path using the `ODO_BOOTSTRAPPER_IMAGE` environment variable.
+
+
+After you push the `odo init image`, you must xref:../../../cli_reference/openshift_developer_cli/using_odo_in_a_restricted_environment/creating-and-deploying-a-component-to-the-disconnected-cluster.adoc#mirroring-a-supported-builder-image_creating-and-deploying-a-component-to-the-disconnected-cluster[mirror a supported builder image] from the registry, xref:../../../cli_reference/openshift_developer_cli/using_odo_in_a_restricted_environment/creating-and-deploying-a-component-to-the-disconnected-cluster.adoc#overwriting-the-mirror-registry_creating-and-deploying-a-component-to-the-disconnected-cluster[overwrite a mirror registry] and then xref:../../../cli_reference/openshift_developer_cli/using_odo_in_a_restricted_environment/creating-and-deploying-a-component-to-the-disconnected-cluster.adoc#creating-a-nodejs-application-with-odo_creating-and-deploying-a-component-to-the-disconnected-cluster[create your application]. A builder image is necessary to configure a runtime environment for your application, it also contains the build tool needed to build your application, for example npm for Node.js or Maven for Java. A mirror registry contains all the necessary dependencies for your application.
```

Comment:
This is incomplete: `A builder image is necessary to configure a runtime environment for your application, it also contains ...`.

Suggest `and also contains`